osv/ecosystems/_ecosystems.py

```diff
@@ -105,27 +108,24 @@
 def get(name: str) -> Ecosystem:
   """Get ecosystem helpers for a given ecosystem."""
 
-  if name.startswith('Debian:'):
-    return Debian(name.split(':')[1])
+  if name.startswith('Debian'):
+    return Debian(name.partition(':')[2])
 
   if name.startswith('Alpine:'):
-    return Alpine(name.split(':')[1])
+    return Alpine(name.partition(':')[2])
 
-  if name.startswith('AlmaLinux:'):
-    # TODO(unassigned)
-    return OrderingUnsupportedEcosystem()
+  if name.startswith('AlmaLinux'):
+    return AlmaLinux()
 
-  if name.startswith('Rocky Linux:'):
-    # TODO(unassigned)
-    return OrderingUnsupportedEcosystem()
+  if name.startswith('Rocky Linux'):
+    return RockyLinux()
 
   if name.startswith('Photon OS:'):
     # TODO(unassigned)
     return OrderingUnsupportedEcosystem()
 
-  if name.startswith('Ubuntu:'):
-    # TODO(unassigned)
-    return OrderingUnsupportedEcosystem()
+  if name.startswith('Ubuntu'):
```

Comment:
nit (and not really important): removing the `:` from all these `startswith` checks makes things like `Ubuntuabcd` `Debian!!!`, `Rocky Linux is an ecosystem`, etc. match these checks.

Maybe you could do something like `base_name, sep, version = name.partition(':')` first, and then check if `base_name` equals the ecosystem names

Reply:
And for a future refactor, all `Ecosystem` helpers can take a `version` in their constructors, and this whole function can just become something like:
```python
def get(name: str) -> Ecosystem:
  base_name, _, version = name.partition(':')
  return _ecosystems.get(base_name)(version)
```

Reply:
I'll do this in a follow up PR where I will add some checks for invalid ecosystem such as query for Alpine without release number.